cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -268,7 +268,7 @@ func getControllerManagerCommand(cfg *kubeadmapi.ClusterConfiguration, k8sVersio
 	caFile := filepath.Join(cfg.CertificatesDir, kubeadmconstants.CACertName)
 
 	defaultArguments := map[string]string{
-		"bind-address":                     "127.0.0.1",
+		"bind-address":                     "0.0.0.0",
```

Comment:
I'd prefer not to change to `0.0.0.0`, which is too open for all tcp connections.
A more doable way is to provide an option, like `bind-address`, in current `ControlPlaneComponent` struct.

Reply:
+1